AutoClicker/Views/MainWindow.xaml.cs

```diff
@@ -218,6 +218,9 @@ private int CalculateInterval()
         {
             var fixedInterval = CalculateFixedInterval();
 
+            // Update IsRandomizedIntervalEnabled
+            AutoClickerSettings.IsRandomizedIntervalEnabled = fixedInterval == 0;

```

Comment:
@bnn678 
Try putting a breakpoint here and debug the program. See if it stops here with different results from what you expect.

Reply:
The value is what I expect, but the binding does not appear to update. I'm not sure why

Reply:
Maybe you should use a different UpdateSourceTrigger

Reply:
I'm not sure how to implement Explicit and the others don't work.

![image](https://user-images.githubusercontent.com/23563750/155044902-80e5812c-751c-4235-b0d0-e59661b61fc9.png)

Reply:
Turns out it isn't that difficult to explicitly call the trigger (though it is certainly more messy). Perhaps I was always supposed to call this method for the trigger? Not sure. Regardless, I guess the latest is the working implementation.